setup.cfg

```diff
@@ -30,6 +30,7 @@ install_requires =
 test =
     pytest
     pytest-cov
+    importlib-metadata; python_version < "3.8"
```

Comment:
I think this needs to be an `install_requires` entry rather than an optional `test` extra.

Reply:
Correct.

I suppose it's worth mentioning that support for python versions < 3.8 (i.e 3.7 and earlier) is ending in a couple of months (2023-06-27).

Reply:
Indeed.  And for comparison: NEP-29's more aggressive schedule has already dropped 3.7 and will drop 3.8 on 2023-04-14!